src/environmentbase/networkbase.py

```diff
@@ -63,24 +89,14 @@ def construct_network(self):
                         ToPort='22',
                         IpProtocol='tcp',
                         CidrIp=FindInMap('networkAddresses', 'vpcBase', 'cidr'))]))
-        self.template.add_output(Output('commonSecurityGroupId', Value=self.template.common_security_group))
+        self.add_output(Output('commonSecurityGroupId', Value=self.common_security_group))
 
         for x in range(0, az_count):
-            self.template._azs.append(FindInMap('RegionMap', Ref('AWS::Region'), 'az' + str(x) + 'Name'))
-
-    def create_action(self):
-        """
-        Override EnvironmentBase.create_action() to construct the network components before the create_hook()
-        """
-        self.load_config()
-        self.initialize_template()
-        self.construct_network()
-        self.create_hook()
-        self.serialize_templates()
+            self._azs.append(FindInMap('RegionMap', Ref('AWS::Region'), 'az' + str(x) + 'Name'))
 
     def add_vpc_az_mapping(self,
                            boto_config,
-                           az_count=2):
+                           az_count=2, cached=False):
```

Comment:
I was also thinking of adding a new `config.json` parameter, but decided to keep the changes to a minimum for now